src/map/lua/luautils.cpp

```diff
@@ -2714,18 +2714,18 @@ namespace luautils
                     }
 
                     CLuaBaseEntity LuaMobEntity(PMob);
-                    CLuaBaseEntity LuaKillerEntity(PChar);
                     CLuaBaseEntity LuaAllyEntity(PMember);
-
+                    bool isKiller = (PMember == PChar) ? true : false;
```

Comment:
@teschnei hadn't actually see it yet just woke aup. And yeah I dunno why I did that the way I did...